tests/foundationals/fuyu/test_fuyu.py

```diff
@@ -0,0 +1,130 @@
+import random
+from pathlib import Path
+from typing import List
+from warnings import warn
+
+import pytest
+import torch
+from PIL import Image
+from torch import Tensor, device as Device
+from transformers import FuyuForCausalLM, FuyuProcessor  # type: ignore[reportMissingTypeStubs]
+
+from refiners.fluxion.utils import load_from_safetensors, manual_seed, no_grad, tensor_to_image
+from refiners.foundationals.fuyu.fuyu import Fuyu, Fuyu8b
+
+
+def _img_open(path: Path) -> Image.Image:
+    return Image.open(path).convert("RGB")  # type: ignore
+
+
+@pytest.fixture(scope="module")
+def our_model(test_weights_path: Path, test_vocab_path: Path, test_device: Device) -> Fuyu:
```

Comment:
the `test_vocab_path` is missing

Reply:
I added the fixture, assuming having a "test_vocab" folder in the test fuyu folder. Perhaps you would prefer having something more generic added to the conftest.py ?